pvl/encoder.py

```diff
@@ -80,23 +91,48 @@ def __init__(self, grammar=None, decoder=None,
 
         # This list of 3-tuples *always* has our own pvl quantity object,
         # and should *only* be added to with self.add_quantity_cls().
-        self.quantities = [(Units, 'value', 'units')]
+        self.quantities = [(Quantity, 'value', 'units')]
```

Comment:
although tuples work, it would be more readable if a quantity dataclass (or named tuple) was used, later you access these attributes by index in the tuple so it is unclear what is going on without having this prototype in ones head

Reply:
Fair enough, this was always meant to be used only inside this module, but adding a namedtuple to contain these three elements does make the code more readable.  Fixed.